modules/psap-special-resource-operator-using.adoc

```diff
@@ -0,0 +1,378 @@
+
+.Prerequisites
+
+* You have a running {product-title} cluster
+* You have installed the OpenShift CLI (`oc`).
+* You are logged into the OpenShift CLI as a user with `cluster-admin` privileges.
+* You have installed the xref:../psap-node-feature-discovery#installing-the-node-feature-discovery-operator_node-feature-discovery-operator[Node Feature Discovery (NFD) Operator]
+* You have installed the Special Resource Operator.
+
+.Procedure
+. To deploy the simple-kmod using the SRO image's local Helm repository, the following SpecialResource manifest can be used. Save this YAML as `simple-kmod-local.yaml`
++
+[source,yaml]
+----
+apiVersion: sro.openshift.io/v1beta1
+kind: SpecialResource
+metadata:
+  name: simple-kmod
+spec:
+  namespace: simple-kmod
+  chart:
+    name: simple-kmod
+    version: 0.0.1
+    repository:
+      name: example
+      url: file:///charts/example
+  set:
+    kind: Values
+    apiVersion: sro.openshift.io/v1beta1
+    kmodNames: ["simple-kmod", "simple-procfs-kmod"]
+    buildArgs:
+    - name: "KMODVER"
+      value: "SRO"
+  driverContainer:
+    source:
+      git:
+        ref: "master"
+        uri: "https://github.com/openshift-psap/kvc-simple-kmod.git"
+----
+
+. Create the SpecialResource with:
++
+[source,terminal]
+----
+$ oc create -f simple-kmod-local.yaml
+----
+
+. The simple-kmod resources will be deployed in the namespace `simple-kmod` as specified in the object manifest. After a short time, the build Pod for the simple-kmod driver container should start running. After a few minutes, the build should complete and the driver container Pods should start Running.
++
+[source,terminal]
+----
+$ oc get pods -n simple-kmod
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                  READY   STATUS      RESTARTS   AGE
+simple-kmod-driver-build-12813789169ac0ee-1-build     0/1     Completed   0          7m12s
+simple-kmod-driver-container-12813789169ac0ee-mjsnh   1/1     Running     0          8m2s
+simple-kmod-driver-container-12813789169ac0ee-qtkff   1/1     Running     0          8m2s
+----
+
+. The logs of the simple-kmod driver container image build can be displayed by getting the logs of the build Pod. For example, using the Pod name from the previous command.
++
+[source,terminal]
+----
+$ oc logs pod/simple-kmod-driver-build-12813789169ac0ee-1-build -n simple-kmod
+----
+
+. To verify that the simple-kmod kernel modules are loaded, execute the `lsmod` command in one of the driver container Pods from step 3. 
++
+[source,terminal]
+----
+$ oc exec -n simple-kmod -it pod/simple-kmod-driver-container-12813789169ac0ee-mjsnh -- lsmod | grep simple
+----
++
+.Example output
+[source,terminal]
+----
+simple_procfs_kmod     16384  0
+simple_kmod            16384  0
+----
+
+To remove the simple-kmod kernel module from the node, simple delete the SpecialResource created in step 2. The kernel module will be unloaded upon deletion of the driver-container Pod.
```

Comment:
How do you delete the SpecialResource? Do you delete the simple-kmod-local.yaml file? If so, maybe:

To remove the simple-kmod kernel module from the node, delete the simple-kmod-local.yaml file that contains the SpecialResource specification. The kernel module is unloaded when the driver container pod is deleted.

Reply:
I rewrote this to be more clear.